zetasql/analyzer/resolver_test.cc

```diff
@@ -718,7 +718,7 @@ TEST_F(ResolverTest, TestResolveCastExpression) {
   ResolveFunctionFails("CAST(1 as BIGINT)", "Type not found: BIGINT");
   ResolveFunctionFails("CAST(1 as SMALLINT)", "Type not found: SMALLINT");
   ResolveFunctionFails("CAST(1 as real)", "Type not found: real");
-  ResolveFunctionFails("CAST(1 as NUMBER)", "Type not found: NUMBER");
+//   ResolveFunctionFails("CAST(1 as NUMBER)", "Type not found: NUMBER");
```

Comment:
Interesting that there was a test for this type.

Reply:
It seems that these types are not supported in Google Standard SQL.